### ISSUE TITLE
Distribute App error

### ISSUE THREAD

**Issue by rafaelviegas86** [aws/amazon-chime-sdk-ios#57]
Hello! After I finished my project and tried to send to TestFlight, I got the following error: 

**Code signing "AmazonChimeSDK.framework" failed.**

I did all suggested steps:
1. Imported SDK to my project like explained in Getting Started 
2. Developed in Xcode and tested. Everything worked fine.
3. Built a new version of my app and clicked in "Distribute App", in Xcode, hoping to send it to Apple. In the last step I got the error: **Code signing "AmazonChimeSDK.framework" failed."**

Usually I don't have problems at that point, I just send to Apple and can manage the new version in itunes connect, to use in test flight or in a future release. Could you help me with this issue?

- Version AmazonChimeSDK: [0.6.1]
- Version AmazonChimeSDKMedia: [0.4.3]

**Comment by haifengx** [comment]
Can you check if this proposed solution can fix the issue: https://stackoverflow.com/a/53267826. Basically you will need to run a script in Build phases to clean up unused architectures.

**Comment by rafaelviegas86** [comment]
Hello, thanks for your advise. In fact, after I posted this question, I googled it more and found a simular solution in: https://stackoverflow.com/questions/42641806/check-and-remove-unsupported-architecture-x86-64-i386-in-ipa-archive

With that script, I was able to send my app to App Store Connect! I was not sure if it was the best solution, but your answer now confirms it. I'm still waiting my review in TestFlight, after I download and try it, I will send a feedback here as soon as possible.
